src/BuiltInTools/dotnet-watch/HotReload/RudeEditDialog.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
```

Comment:
is this just a rename showing up as a whole new file?

Reply:
Primarily yes, the file name didn't match the class name so I updated. Some logic was moved to the ConsoleRequester.